model/change_point_detection.go

```diff
@@ -104,10 +105,11 @@ func (ts TriageStatus) Validate() error {
 }
 
 type PerformanceResultSeriesID struct {
-	Project string `bson:"project"`
-	Variant string `bson:"variant"`
-	Task    string `bson:"task"`
-	Test    string `bson:"test"`
+	Project     string `bson:"project" json:"project"`
+	Variant     string `bson:"variant" json:"variant"`
+	Task        string `bson:"task" json:"task"`
+	Test        string `bson:"test" json:"test"`
+	ThreadLevel int32  `bson:"thread_level" json:"thread_level"`
```

Comment:
this seems like we're collapsing information across some kind of boundary

also thread_level isn't a top level concept in cedar's data model, and I think it might make sense to rethink this a bit.

Reply:
@tychoish Q: do perf results have different thread levels for the measurements being made under them?

In any case, thread level should be applied to the individual measurement time series/change points.

Reply:
What do you mean "applied" we don't treat thread level different from any other parameter passed to the test. It is captured but not specially.